packages/expo/src/Haptic/Haptic.ts

```diff
@@ -45,7 +45,9 @@ export enum ImpactFeedbackStyle {
 /**
  * Triggers notification feedback.
  */
-export function notification(type: NotificationFeedbackType = NotificationFeedbackType.Success) {
+export async function notification(
```

Comment:
btw (not for this PR) - might be good to change the native side of the haptic functions to be async as well (the promise resolves when we successfully make a call to the haptic library) because even with turbomodules, we'll need to `dispatch_async` from the JS queue to the main queue since UIFeedbackGenerator is a main-threaded UI* class